lib/cryptoexchange/exchanges/f1cx/services/market.rb

```diff
@@ -0,0 +1,38 @@
+module Cryptoexchange::Exchanges
+  module F1cx
+    module Services
+      class Market < Cryptoexchange::Services::Market
+        class << self
+          def supports_individual_ticker_query?
+            true
```

Comment:
You can actually do it in a single query with https://f1cx.com/api/v2/tickers
we can save api calls
@wenjieyek